packages/grid/x-data-grid/src/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -262,9 +262,9 @@ export const useGridVirtualScroller = () => {
         break;
     }
 
-    if (innerSize.width === 0) {
```

Comment:
This condition makes sense, but it's always 0 in JSDOM.
I assume we didn't have a check like this before, do we really need it?

Reply:
Let's see if the tests pass without it. The state dimensions are now always defined but initialized with `0`, so this condition might have been added to avoid the uninitialized case, which may have other implications. If it works without it, amazing. Otherwise, we could compare `innerSize !== EMPTY_SIZE`, which is the initiali value for all dimensions.

Reply:
All tests pass now - see [`1a925a6` (#10059)](https://github.com/mui/mui-x/pull/10059/commits/1a925a6b3af42c16677f31c2bab72b537b002ab4) and [`af2f722` (#10059)](https://github.com/mui/mui-x/pull/10059/commits/af2f722262041aef9b8cff4d531da4473a241665)